WcaOnRails/config/locales/en.yml

```diff
@@ -1665,72 +1665,89 @@ en:
+          '1': "The location of where competitions are held is based on where there are people who want to organize. If you are interested in organizing a competition, it's highly recommended to attend at least one or two competitions as a competitor to learn from the experience."
+          '2': "WCA Competitions must follow the <a target='_blank' href='https://www.worldcubeassociation.org/regulations/'>WCA Regulations</a>. The organization team doesn't have to know the regulations by heart, however, a basic knowledge of the regulations would definitely help in correctly organizing the competition. It is suggested that potential organizers scope out venue options and get an idea for some dates when it would be best to hold the competition in their local area."
           '3': "After that, the organization team must contact a nearby <a target='_blank' href='https://www.worldcubeassociation.org/delegates'>WCA Delegate</a>. The supervision of a WCA Delegate is required in order to make a competition officially sanctioned by the WCA. The WCA Delegate will help guide the organization on how to host a successful competition."
           '4': "Visit the <a target='_blank' href='https://www.worldcubeassociation.org/organizer-guidelines'>WCA Competition Organizer Guidelines</a> for further information."
       '6':
         title: "What are the WCA accounts for? What is the difference between WCA accounts and WCA profiles?"
         #context: Paragraph of the answer to "What are the WCA accounts for? ..."
         content:
-          '1': "You are assigned a WCA ID when the results from your first WCA sanctioned competition are published."
-          '2': "WCA accounts allow you to prove to us who you are. This allows you to add a picture to your WCA profile, and allows you to register for competitions. This will protect you from other people pretending to be you."
+          '1': "You are assigned a WCA ID when the results from your first WCA sanctioned competition are published. The WCA ID will be associated with your WCA Profile page and will be tied to all of our results."
+          '2': "WCA accounts are tied to your preferred email address and allow you to prove to us who you are. This allows you to add a picture to your WCA profile, and allows you to register for competitions. This will protect you from other people pretending to be you."
+          '3': "The account you use to register for your first WCA competition will be automatically connected with your WCA ID and profile once the results have been submitted from the competition. Be sure to use an email address that you check often so that any competition communications will reach you when needed."
           #context: Yup, nothing to translate here! Just hit "Use the original version"
-          '3': "%{current_user_status}"
+          '4': "%{current_user_status}"
```

Comment:
Same issue with the key: could you please keep `3` here and change the key above to something else?